extra/intl-extra/IntlExtension.php

```diff
@@ -241,6 +247,36 @@ public function getCountryTimezones(string $country): array
             return [];
         }
     }
+    
+    public function getLanguageNames(string $locale = null): array
+    {
+        return Languages::getNames($locale);
```

Comment:
This must handle the MissingResourceException as done in filters

Reply:
For instance?
```php
public function getLanguageNames(string $locale = null): array
{
    try {
        return Languages::getNames($locale);
    } catch (MissingResourceException $exception) {
        return [];
    }
}
```

Reply:
yes